com.unity.cinemachine/Documentation~/CinemachineBindingModes.md

```diff
@@ -83,7 +83,7 @@ The offset is interpreted in world space relative to the origin of the Follow ta
 
 Simple follow with world up interprets the offset and damping values in camera-local space. This mode emulates the action a human camera operator would take when instructed to follow a target.
 
-The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regard to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it.
+The camera attempts to move as little as possible to maintain the same distance from the target; the direction of the camera with regards to the target does not matter. Regardless of the orientation of the target, the camera tries to preserve the same distance and height from it.
```

Comment:
Hmmm, this one is debatable.  Maybe use a different expression altogether?
https://prowritingaid.com/with-regards-to-vs-with-regard-to#:~:text=With%20Regards%20To-,With%20Regard%20To%20or%20With%20Regards%20To%3F,or%20%E2%80%9Cwith%20relation%20to.%E2%80%9D

Reply:
Hmmm after reading again think you are right and probably should remain as regard. I will rework this one a bit.